tests/cli/test_cmd_lock.py

```diff
@@ -0,0 +1,194 @@
+
+import click.testing
+
+from sortinghat.cli.client import SortingHatClientError
+from sortinghat.cli.cmds.lock import add, rm
+
+
+LOCK_ADD_CMD_OP = """mutation {{
+  lockIdentity(uuid: "{}") {{
+    uuid
+  }}
+}}"""
+
+LOCK_RM_CMD_OP = """mutation {{
+  unlockIdentity(uuid: "{}") {{
+    uuid
+  }}
+}}"""
+
+
+LOCK_ADD_OUTPUT = """Unique identity {} is locked\n"""
+
+LOCK_RM_OUTPUT = """Unique identity {} is unlocked\n"""
+
+
+LOCK_NOT_FOUND_ERROR = (
+    "FFFFFFFFFFFFFFF not found in the registry"
+)
+
+
+class MockClient:
+    """Mock client"""
+
+    def __init__(self, responses):
+        self.responses = responses
+        self.ops = []
+
+    def connect(self):
+        pass
+
+    def disconnect(self):
+        pass
+
+    def execute(self, operation):
+        self.ops.append(operation)
+        response = self.responses.pop(0)
+
+        if isinstance(response, SortingHatClientError):
+            raise response
+        else:
+            return response
+
+
+class TestLockAddCommand(unittest.TestCase):
+    """Lock add command unit tests"""
+
+    def test_lock_add(self):
+        """Check if it adds a lock to a unique identity"""
+
+        responses = [
+            {'data': {'lockIdentity': {'uuid': '322397ed782a798ffd9d0bc7e293df4292fe075d'}}}
+        ]
+        mock_client = MockClient(responses)
+
+        runner = click.testing.CliRunner()
+
+        # Add lock
+        params = [
+            '322397ed782a798ffd9d0bc7e293df4292fe075d'
+        ]
+        result = runner.invoke(add, params, obj=mock_client)
+
+        expected = LOCK_ADD_CMD_OP.format('322397ed782a798ffd9d0bc7e293df4292fe075d')
+        self.assertEqual(len(mock_client.ops), 1)
+        self.assertEqual(str(mock_client.ops[0]), expected)
+
+        expected = LOCK_ADD_OUTPUT.format('322397ed782a798ffd9d0bc7e293df4292fe075d')
+        self.assertEqual(result.stdout, expected)
+        self.assertEqual(result.exit_code, 0)
+
+    def test_error(self):
+        """"Check if it fails when an error is sent by the server"""
+
+        error = {
+            'message': LOCK_NOT_FOUND_ERROR,
+            'extensions': {
+                'code': 9
+            }
+        }
+
+        responses = [
+            SortingHatClientError(error['message'], errors=[error])
+        ]
+        mock_client = MockClient(responses)
+
+        runner = click.testing.CliRunner(mix_stderr=False)
+
+        params = [
+            'FFFFFFFFFFFFFFF'
+        ]
+        result = runner.invoke(add, params, obj=mock_client)
+
+        expected = LOCK_ADD_CMD_OP.format('FFFFFFFFFFFFFFF')
+        self.assertEqual(len(mock_client.ops), 1)
+        self.assertEqual(str(mock_client.ops[0]), expected)
+
+        expected_err = "Error: " + LOCK_NOT_FOUND_ERROR + '\n'
+        self.assertEqual(result.stderr, expected_err)
+        self.assertEqual(result.exit_code, 9)
+
+
+class TestLockRmCommand(unittest.TestCase):
+    """Lock rm command unit tests"""
+
+    def test_lock_add(self):
```

Comment:
I think this should be renamed to `test_lock_rm`.

Reply:
Fixed. Please check again.